services/simcore/.env

```diff
@@ -6,56 +6,60 @@
 BF_API_KEY=none
 BF_API_SECRET=none
 
-CATALOG_HOST=${PREFIX_STACK_NAME}_catalog
+CATALOG_HOST=${SIMCORE_STACK_NAME}_${PREFIX_STACK_NAME}_catalog
 
-DIRECTOR_HOST=${PREFIX_STACK_NAME}_director
+DIRECTOR_HOST=${SIMCORE_STACK_NAME}_${PREFIX_STACK_NAME}_director
 
-DOCKER_IMAGE_TAG=staging-latest
+DOCKER_IMAGE_TAG=master-latest
 
 EXTRA_HOSTS_SUFFIX=osparc.local
 
 OPS_MONITORED_NETWORK=monitored_network
 OPS_PUBLIC_NETWORK=public-network
 
 POSTGRES_DB=simcoredb
-POSTGRES_ENDPOINT=${PREFIX_STACK_NAME}_postgres:5432
-POSTGRES_HOST=${PREFIX_STACK_NAME}_postgres
-
+POSTGRES_ENDPOINT=local_postgres:5432
+POSTGRES_HOST=local_postgres
```

Comment:
can we please then keep it as? so it is still compatible with the deployer in case of // deployment? You anyway replace them with the psed afterwards. this way it works with both.... same for all the ENVs below...
${PREFIX_STACK_NAME}_postgres:...